doc/classes/Tabs.xml

```diff
@@ -169,7 +184,7 @@
 		<signal name="right_button_pressed">
 			<argument index="0" name="tab" type="int" />
 			<description>
-				Emitted when a tab is right-clicked.
+				Emitted when a tab's right button is pressed. See [method set_tab_button_icon].
```

Comment:
There's also a comment in the `tabs.cpp` (line 143), which has the same mistake (right mouse button, instead of right tab button).

Reply:
Comment updated.